### ISSUE TITLE
Showing raw proba in classifications

### ISSUE THREAD

**Issue by bhaskoro-muthohar** [pycaret/pycaret#945]
I have trained model using lightgbm algorithm for binary classification and fit it to predictions. The predictions dataframe only show Label and it's probability score. 

![image](https://user-images.githubusercontent.com/35159954/102711541-0431ff80-42ed-11eb-847b-469822f9f8e2.png)

I want to see the raw proba of the predictions, how can I do that in PyCaret like in sklearn?

**Comment by Yard1** [comment]
This feature will be present in a future release.

**Comment by bhaskoro-muthohar** [comment]
> This feature will be present in a future release.

For now, let say I want to retrieve the probability of Label class 0, so I just need to do `1-(score of Label 0)` right?

**Comment by Yard1** [comment]
Indeed.

**Comment by Yard1** [comment]
Functionality added in commit `e729cf7d6716bb582233aa4bcf2689e72968b293` - do `predict_model(model, raw_score=True)` to obtain scores for all labels in separate columns.
